webapp-django/crashstats/supersearch/models.py

```diff
@@ -169,7 +169,13 @@ def get(self, **kwargs):
 
 class SuperSearchFields(models.SocorroMiddleware):
 
-    URL_PREFIX = '/supersearch/fields/'
+    @property
+    def implementation(self):
+        if not hasattr(self, '__implementation'):
+            self.__implementation = super_search_fields.SuperSearchFields(
+                config=config_from_configman()
+            )
+        return self.__implementation
```

Comment:
I'm pretty sure there's a way to have this method in ``models.SocorroMiddleware`` and to have something simpler here, like ``implementation_class = super_search_fields.SuperSearchFields``. For the sake of the glory DRY. :)